src/popups/ConfirmWindow/ConfirmWindow.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { confirmable } from 'react-confirm';
+import classNames from 'classnames';
+
+import styles from './ConfirmWindow.module.css';
+
+export const ConfirmWindow = confirmable(({ show, proceed, conformation }) => {
```

Comment:
ConfirmModal? ConfirmDialog? ConfirmPopup?
Window is a too general name here IMHO